.cspell.json

```diff
@@ -1128,7 +1129,9 @@
     "pyyaml",
     "qdisc",
     "qframe",
+    "QImage",
     "qlen",
+    "QObject",
```

Comment:
I believe you should change the variable names. `QImage` and `QObject` don't cause errors.
![image](https://user-images.githubusercontent.com/31987104/188578494-965be5ab-3faa-4094-b031-4020d947c315.png)

```suggestion
    "qlen",
```

Reply:
Oh...
I'll delete qimage and fix variables.
However, qobject is included at `qobject_cast`, so I'll fix QObject to qobject